com.makemkv.MakeMKV.yaml

```diff
@@ -9,6 +9,8 @@ finish-args:
   # X11 access
   - --socket=x11
```

Comment:
If I understand the documentation correctly, we should also add `--socket=fallback-x11` then.

Reply:
I believe switching `x11` to `fallback-x11` is commonly used when enabling Wayland support:

https://github.com/search?q=org%3Aflathub%20--socket%3Dfallback-x11&type=code